app/gcl/associate-cloud-engineer/section3/NavBar.tsx

```
@@ -0,0 +1,100 @@
+            }
+        });
+
+        return () => observer.disconnect();
+    }, []);
+
+    return (
+        <nav className="sidebar" aria-label="Section 3 ガイドナビゲーション">
+            {NAV_ITEMS.map((item, idx) => {
+                if (item.type === 'title') {
+                    return (
+                        <div key={`title-${idx}`} className="nav-section-title">
```

Biome (2.4.16)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)